src/rgw/rgw_cr_rados.cc

```diff
@@ -635,7 +635,7 @@ int RGWAsyncFetchRemoteObj::_send_request()
 
   char buf[16];
   snprintf(buf, sizeof(buf), ".%lld", (long long)store->getRados()->instance_id());
```

Comment:
this isn't new, but should we use the explicit integral notations in places like this, and also explicit integral types?